components/Input/Input.themes.ts

```diff
@@ -14,25 +15,25 @@ export namespace Theme {
     inputInvalidBorder: Property.Color;
   };
 
-  type Factory = (primaryColor?: Property.Color) => Colors;
+  type Factory = (primaryColor: ColorInfo) => Colors;
 
-  export const getLight: Factory = () => ({
+  export const getLight: Factory = (primaryColor) => ({
     inputBg: '$deepBlue1',
     inputBorder: '$grayBlue9',
     inputFocusBg: tinycolor('black').setAlpha(0.15).toHslString(),
-    inputFocusBorder: '$neon8',
+    inputFocusBorder: `$${primaryColor.name}8`,
```

Comment:
Sounds a lot like choosing a hue for a color, what about a helper ?
```
type HueValue: '1' | '2' | '3' | '4' | '5' | '6' | '7' | '8' | '9' | '10' | '11' | '12' | 'A1' | A'2' | 'A3' | 'A4' | 'A5' | 'A6' | 'A7' | 'A8' | 'A9' | 'A10' | 'A11' | 'A12'
type Hue: (primaryColor: ColorInfo, value: HueValue) => Property.Color

const hue: Hue = ({name}, value) => `$${name}${value}`
```

Reply:
Tbh I'm not finding having this function simplifying it:
```javascript
bc: `$${primaryColor.name}8`
```
vs 

```javascript
import pickHue from 'utils/pickHue';
...
bc: pickHue({ name: primaryColor}, 8),
```

I'm going to find a way to have this instead:

```javascript
bc: pickPrimaryScale(8),
```